### ISSUE TITLE
this.props.stop(): Is there a copilot function that stops the tutorial that we could use? 

### ISSUE THREAD

**Issue by ernymarvind** [okgrow/react-native-copilot#74]
Is there a way to stop copilot without clicking the buttons from `TooltipComponent` (handleStop function)?

I would like to call `handleStop` or similar function and stop copilot before the app goes to the background/foreground.

**Comment by mohebifar** [comment]
Is `this.props.stop` in the wrapped component what you're looking for?

**Comment by ernymarvind** [comment]
yes, it is. But how do I use it on the parent component like how we start copilot on the main screen using `this.props.start()`?

Calling `this.props.stop()` in the main component crashes the app. It does not have .stop() function when logging the props. 

`'PROPS', { 
    screenProps: undefined,
	navigation: { 
        dispatch: [Function],
        state: { routeName: 'TasksAndGoals', key: 'id-1536975371061-1' 
    },
        isFocused: [Function: isFocused],
        addListener: [Function],
        goBack: [Function: goBack],
        navigate: [Function: navigate],
        pop: [Function: pop],
        popToTop: [Function: popToTop],
        setParams: [Function: setParams],
        getParam: [Function: getParam],
        push: [Function: push],
        replace: [Function: replace] 
    },
    start: [Function: _callee4],
    currentStep: null,
    visible: false,
    copilotEvents:{ 
        on: [Function: on],
        off: [Function: off],
        emit: [Function: emit] 
    }
}`

Not sure how to use or retrieve `this.state.stop`.

**Comment by ernymarvind** [comment]
Hi, any updates for `.stop()` on the main component?

**Comment by ernymarvind** [comment]
Had to use it on my custom TooltipComponent and used `handleStop` function instead.

`const TooltipComponent = ({ handleStop, currentStep }) => {
    AppState.addEventListener('change', (nextAppState) => handleStop);
    return(<View />);
}`

**Comment by RichardLitt** [comment]
@mohebifar Does that sound like something we could easily integrate? Is it necessary?
